tools/cldr-apps/js/src/views/MainHeader.vue

```diff
@@ -104,7 +104,6 @@ export default {
         return cldrText.sub("coverage_auto_msg", {
           surveyOrgCov: item.label,
         });
-        return item.label + "—Default";
       }
```

Comment:
Without "—Default" will it still be clear to the user which level is the default?

Reply:
the `coverage_auto_msg` on line 104 is the one that has "… (Default)" in it.

Reply:
great!